### DISCUSSION TITLE
Bounding boxes become too small after fine-tuning PP-OCRv5_server_det

### DISCUSSION THREAD

**Discussion by max2000777** [PaddlePaddle/PaddleOCR#17092] [Q&A]
Hi!

I'm not sure this is the right place to post this, but I'll give it a try.

I'm currently trying to fine-tune PP-OCRv5_server_det.

Here is a very small sample of my dataset: [here](https://drive.google.com/drive/folders/1vxqKWi5CIUJsuZ4fPPOPN0cXsrNhRQEW?usp=sharing). In reality, my dataset contains around 10,000 images (maybe that’s too much?).

<img width="907" height="680" alt="dataset_image1" src="https://github.com/user-attachments/assets/f8655466-949d-4a69-aa58-97fd1e65ecd4" />

The pretrained PP-OCRv5_server_det model produces bounding boxes of the right size on my data, but their placement is okay — which is why I’m trying to fine-tune it.
<img width="768" height="576" alt="Inference exemple 1" src="https://github.com/user-attachments/assets/99c00d2e-eedd-4d71-8028-0a29f6cb9d9f" />

For fine-tuning, I’m using the default configuration file of **PP-OCRv5_server_det** with almost no modification. I only changed the learning rate to 0.0001 (I only have one GPU).

**My issue**: after only two epochs, the model predicts bounding boxes that are a bit too small, cutting off important parts of the text.
<img width="768" height="576" alt="Inference exemple 2" src="https://github.com/user-attachments/assets/763ef50d-104b-43cf-b565-59a36fe6eae7" />

Over the past few days, I tried many things without success:
– enlarging the ground-truth boxes (adding 3 pixels of padding or padding based on the box's length)
– changing PostProcess.unclip_ratio to `1`, `2`, `2.5` 
– training for more epochs
Nothing helped.

The only thing that worked was modifying text_det_unclip_ratio after training, during inference (in the pipeline). But this feels like a workaround rather than a real fix.
One thing I thought would have worked was changing `PostProcess.unclip_ratio` to `2.5` before training.
However, if I understand correctly, even though my new model's configuration file shows `unclip_ratio = 2.5`, the [PaddleX OCR pipeline](https://github.com/PaddlePaddle/PaddleX/blob/release/3.3/paddlex/configs/pipelines/OCR.yaml) overrides it and still forces the default `1.5` ratio, making the change ineffective.

As you might guess from my dataset, my goal is to use this fine-tuned model with [VideOCR](https://github.com/devmaxxing/videocr-PaddleOCR), so I would prefer to avoid drifting too far from the original **PP-OCRv5_server_det** settings.

My questions:
– Should I modify the `Train.transforms`?
– Lower the learning rate even further?
– Reduce the size of my dataset?
Or something else entirely?

I would really appreciate any help, as I’m quite lost at this point.

Thanks in advance!
